docs/admin/install.rst

```diff
@@ -44,6 +45,34 @@ A basic knowledge of the Qubes OS is helpful.
 Pre-install tasks
 -----------------
 
+Verify the SecureDrop server configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In order to be used with SecureDrop Workstation, your instance must be running the latest version of SecureDrop, and the server configuration must have been updated to allow for HTTP ``DELETE`` requests. The configuration change to enable this was added in the ``0.13.0`` version of SecureDrop, released on May 29 2019. If your instance was created using this or a later version, it has the necessary changes. If not, then the ``./securedrop-admin install`` command must have been run from an Admin Workstation updated with the ``0.13.0`` code or later. To check this:
+
+- Use an Admin Workstation USB to boot into Tails, with the persistent volume unlocked and an administration password set.
+- Navigate to **Applications ▸ System Tools ▸ Terminal** to open a terminal.
+- Verify that the *Journalist Interface* Apache configuration allows for HTTP ``DELETE`` using the following command:
+
+  .. code-block:: sh
+
+    ssh app "sudo cat /etc/apache2/sites-enabled/journalist.conf | grep '^  <LimitExcept GET POST HEAD DELETE>$'"
```

Comment:
habit for both, will simplify.

Reply:
As for the regexp, keeping as is  - otherwise it'll match commented-out lines and such.

Reply:
That's fair; if you keep as is, I would add a note about entering the command exactly as is including whitespace though -- easy to enter one space instead of two, unless you copy and paste.

Reply:
Added explicit spaces in regexp instead - best of both worlds!